packages/playground/src/components/Header.tsx

```diff
@@ -67,6 +48,25 @@ function HeaderButtons({ playGroundFormRef }: { playGroundFormRef: React.Mutable
   );
 }
 
+const liveSettingsSchema: RJSFSchema = {
+  type: 'object',
+  properties: {
+    liveValidate: { type: 'boolean', title: 'Live validation' },
+    disable: { type: 'boolean', title: 'Disable whole form' },
```

Comment:
Can you change this to `disabled` since that is the property value in the `Form`